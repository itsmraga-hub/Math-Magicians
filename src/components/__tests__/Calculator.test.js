import React from 'react';
import renderer from 'react-test-renderer';

import CalculatorComponent from '../Calculator';

it('<CalculatorComponent /> Renders properly', () => {
  const tree = renderer.create(<CalculatorComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
