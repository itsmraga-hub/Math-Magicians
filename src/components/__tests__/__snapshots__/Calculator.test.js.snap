// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CalculatorComponent /> Renders properly 1`] = `
<div
  className="calculatorContainer"
>
  <span
    className="grid-span-full grid-item screen"
  />
  <span
    className="grid-span-full grid-item screen"
  />
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    AC
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    +/-
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    %
  </button>
  <button
    className="btn btn-secondary grid-item"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    รท
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    7
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    8
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    9
  </button>
  <button
    className="btn btn-secondary grid-item"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    x
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    4
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    5
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    6
  </button>
  <button
    className="btn btn-secondary grid-item"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    -
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    1
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    2
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    3
  </button>
  <button
    className="btn btn-secondary grid-item"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    +
  </button>
  <button
    className="btn btn-primary grid-item span-2"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    0
  </button>
  <button
    className="btn btn-primary grid-item null"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    .
  </button>
  <button
    className="btn btn-secondary grid-item"
    data-attribute=""
    data-id=""
    onClick={[Function]}
    type="button"
  >
    =
  </button>
</div>
`;
